operations/iuf/workflows/managed_rollout.md

```diff
@@ -0,0 +1,118 @@
+the best approach for rebooting LNet router and gateway nodes outside of IUF that aligns with site preferences.
+
+Once this step has completed:
+
+- Managed LNet router and gateway nodes (if any) have been rebooted to the images and CFS configurations created in previous steps of this workflow
+- Per-stage product hooks have executed for the `managed-nodes-rollout` stage if IUF `managed-nodes-rollout` procedures were used to perform the reboots
+
+### 2.2 Compute nodes
+
+1. Refer to the "Install and Upgrade Framework" section of each individual product's installation documentation to determine if any special actions need to be performed outside of IUF for the `managed-nodes-rollout` stage.
+
+1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage on a single node to ensure the node reboots successfully with the desired image and CFS configuration. This node is
+referred to as the "canary node" in the remainder of this section. Use `--limit-managed-rollout` to target the canary node only and use `-mrs reboot` to reboot the canary node immediately.
+
+    (`ncn-m001#`) Execute the `managed-nodes-rollout` stage with a single xname, rebooting the canary node immediately. Replace the example value of `${XNAME}` with the xname of the canary node.
+
+    ```bash
+    XNAME=x3000c0s29b1n0
+    iuf -a "${ACTIVITY_NAME}" run -r managed-nodes-rollout --limit-managed-rollout "${XNAME}" -mrs reboot
+    ```
+
+1. Verify the canary node booted successfully with the desired image and CFS configuration.
+
+1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage on all nodes, rebooting the nodes in the default staged manner in conjunction with the workload manager. If an
+immediate reboot of compute nodes is desired instead, add `-mrs reboot` to the `iuf run` command.
+
+    (`ncn-m001#`) Execute the `managed-nodes-rollout` stage.
+
+    ```bash
+    iuf -a "${ACTIVITY_NAME}" run -r managed-nodes-rollout
+    ```
+
+Once this step has completed:
+
+- Managed compute nodes have been rebooted to the images and CFS configurations created in previous steps of this workflow
+- Per-stage product hooks have executed for the `managed-nodes-rollout` stage
+
+### 2.3 Application nodes
+
+**`NOTE`** If LNet router or gateway nodes were upgraded in the [2.1 LNet router nodes and gateway nodes](#21-lnet-router-nodes-and-gateway-nodes) section, there is no need to upgrade them again in this section. Follow the
+instructions in this section to upgrade any remaining applications (UANs, etc.) that have not been upgraded yet.
+
+Since applications nodes are not managed by workload managers, the IUF `managed-nodes-rollout` stage cannot reboot them in a controlled manner via the `-mrs stage` argument. The IUF `managed-nodes-rollout` stage can reboot application
```

Comment:
Very minor typo:

```suggestion
Since application nodes are not managed by workload managers, the IUF `managed-nodes-rollout` stage cannot reboot them in a controlled manner via the `-mrs stage` argument. The IUF `managed-nodes-rollout` stage can reboot application
```

Reply:
Fixed in next commit.